src/common/util/include/openvino/util/common_util.hpp

```diff
@@ -14,6 +14,18 @@
 namespace ov {
 namespace util {
 
+// helper struct to tell wheter type T is any of given types U...
+// termination case when U... is empty -> return std::false_type
+template <class T, class... U>
+struct IsAnyOf : public std::false_type {};
```

Comment:
I would prefer snake case for type traits, as it was done in original intel_gpu/.../utils.hpp file.
I think this rule that class should be started with camel case should be for non traits classes

Reply:
There is the CI check
`Code Style / NamingConventionCheck`
which does not allow the `is_any_of` name.

Reply:
May be it is better to move this to intel/cpu/utils folder, for example to cpu_utils.hpp. Thus it should not have an issue with NamingConventionCheck, and we will not have to discuss this changes with other teams, since they will be stored on the plugin level. What do think?

Reply:
@maxnick ok, I added the `is_any_of` to the cpu_utils.hpp